docs/tutorial/02_deploy_sdcore.md

```diff
@@ -45,35 +57,39 @@ deployment by running `juju status`. The core network is ready when all the char
+ausf                               active       1  sdcore-ausf               edge            26  10.152.183.155  no       
+grafana-agent-k8s         0.32.1   waiting      1  grafana-agent-k8s         latest/stable   42  10.152.183.225  no       installing agent
+mongodb-k8s                        active       1  mongodb-k8s               5/edge          36  10.152.183.243  no       Primary
+nms                                active       1  sdcore-nms                edge            12  10.152.183.165  no       
+nrf                                active       1  sdcore-nrf                edge            45  10.152.183.142  no       
+nssf                               active       1  sdcore-nssf               edge            24  10.152.183.127  no       
+pcf                                active       1  sdcore-pcf                edge            19  10.152.183.215  no       
+self-signed-certificates           active       1  self-signed-certificates  beta            33  10.152.183.66   no       
+smf                                active       1  sdcore-smf                edge            24  10.152.183.91   no       
+traefik-k8s               2.9.6    active       1  traefik-k8s               latest/stable  129  10.0.0.3        no       
+udm                                active       1  sdcore-udm                edge            22  10.152.183.171  no       
+udr                                active       1  sdcore-udr                edge            20  10.152.183.84   no       
+upf                                active       1  sdcore-upf                edge            47  10.152.183.240  no       
+webui                              active       1  sdcore-webui              edge            15  10.152.183.109  no       
+
+Unit                         Workload  Agent  Address       Ports  Message
+amf/0*                       active    idle   10.1.194.235         
+ausf/0*                      active    idle   10.1.194.220         
+grafana-agent-k8s/0*         blocked   idle   10.1.194.230         grafana-cloud-config: off, logging-consumer: off
+mongodb-k8s/0*               active    idle   10.1.194.241         Primary
+nms/0*                       active    idle   10.1.194.242         
+nrf/0*                       active    idle   10.1.194.195         
+nssf/0*                      active    idle   10.1.194.199         
+pcf/0*                       active    idle   10.1.194.204         
+self-signed-certificates/0*  active    idle   10.1.194.247         
+smf/0*                       active    idle   10.1.194.224         
+traefik-k8s/0*               active    idle   10.1.194.226         
+udm/0*                       active    idle   10.1.194.212         
+udr/0*                       active    idle   10.1.194.203         
+upf/0*                       active    idle   10.1.194.193         
+webui/0*                     active    idle   10.1.194.198
 ```
```

Comment:
We'll also want to use  the NMS for configuring instead of the curl's, do we want to place that in a different PR?

Reply:
No, it makes sense to have it here.
Done.